core/app/modules/kit-library/assets/js/data/component.js

```diff
@@ -1,11 +1,14 @@
-import * as dataCommands from './commands';
+import { Index, KitsDownloadLink } from './commands';
```

Comment:
```suggestion
import * as commandsData from './commands-data/';
```
https://github.com/elementor/elementor/blob/master/docs/core/common/assets/js/api/core/commands.md#user-content-guidelines-conventions--files-structure